bench/tests/test_init.py

```diff
@@ -57,11 +57,10 @@ def test_multiple_benches(self):
 		})
 
 	def test_new_site(self):
+		self.init_bench('test-bench')
 		self.new_site("test-site-1.dev")
 
 	def new_site(self, site_name):
-		self.test_init()
```

Comment:
Why remove this? The only problem was that `archived_sites_path` variable was missing